panther_lights/CMakeLists.txt

```diff
@@ -9,35 +9,85 @@ find_package(ament_cmake REQUIRED)
 find_package(image_transport REQUIRED)
 find_package(panther_gpiod REQUIRED)
 find_package(panther_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(sensor_msgs REQUIRED)
+find_package(yaml-cpp REQUIRED)
 
 include_directories(include)
 
+pluginlib_export_plugin_description_file(${PROJECT_NAME} plugins.xml)
+
+# Create list with animation plugins
```

Comment:
Why use a list? Even if there are more than two animations, it's easier to move them to add_library without a list. Are there any issues without using a list?